### PR TITLE
Updated data-system.md

### PR DESCRIPTION
**Pull request by HennaSheikh**
Edited for tone, consistency, and general copy errors.

### CODE REVIEW

**Comment by HennaSheikh** [comment]
I have signed and submitted the CLA.

File: app/1.0/docs/devguide/data-system.md [hunk 1]
```diff
@@ -1,195 +1,117 @@
----
-title: Data system concepts
----
```
Comment (arthurevans) [inline app/1.0/docs/devguide/data-system.md]: This metadata is required.

File: app/1.0/docs/devguide/data-system.md [hunk 2]
```diff
@@ -1,195 +1,117 @@
...
+Polymer allows you to observe changes to an element's properties and take various actions based on data changes. These actions, or property effects, include:
 
-<!-- toc -->
```
Comment (arthurevans) [inline app/1.0/docs/devguide/data-system.md]: This comment instructs the templating system to add a generated table contents.

File: app/1.0/docs/devguide/data-system.md [hunk 3]
```diff
@@ -1,195 +1,117 @@
...
+Observers. Callbacks invoked when data changes.
 
+Computed properties. Virtual properties which are computed based on other properties, and recomputed when the input data changes.
 
-Polymer lets you observe changes to an element's properties and take various actions based on data
-changes. These actions, or *property effects*, include:
+Data bindings. Annotations that update the properties, attributes, or text content of a DOM node when data changes.
 
-
-*   Observers. Callbacks invoked when data changes.
```
Comment (arthurevans) [inline app/1.0/docs/devguide/data-system.md]: This is markdown formatting for a bulleted list.

**Comment by arthurevans** [comment]
I think you've misunderstood the format for these docs. You've removed all of the Markdown formatting, images, and metadata. The diffs are so large I can't tell whether you've made any actual editorial changes, but the changes I've looked at are simply stripping the doc of its formatting. Closing.

**Comment by HennaSheikh** [comment]
My apologies, Arthur.  I think I know what went wrong when porting the plain text back to GitHub.  I'll try it again, but with each edit (or chunk of edits) as its own smaller request.  I'm sorry to have made such a mess of that!

**Comment by arthurevans** [comment]
No worries. Thanks!

**Pull request closed without merging**